lms/services/vitalsource/client.py

```diff
@@ -77,32 +80,39 @@ def parse_document_url(document_url):
     def generate_document_url(book_id, cfi):
         return f"vitalsource://book/bookID/{book_id}/cfi/{cfi}"
 
-    def get_launch_params(self, document_url, lti_user: LTIUser) -> Tuple[str, Optional[Dict[str, str]]]:
+    def get_iframe_url(self, document_url: str) -> str:
+        """
+        Return an URL like https://hypothesis.vitalsource.com/books/{book_id}/cfi/{cfi}.
```

Comment:
The exact URL is an implementation detail that will probably change in future, we're still discussing that with them. I would say something like "Return a URL to load the VitalSource book viewer at a particular book and location."